### DISCUSSION TITLE
Work with live stream chat

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#543] [Ideas]
Does this program work even on live stream chats to detect spam? Big content creators, delete their live streams most of the time, but if they dont, those spam messeges are annoying for people viewing stream, and there is a new scam recently started, so if it works for live stream chat, that will be great too.

New scam: https://www.youtube.com/watch?v=p_UF-2Z3cMc

**Comment by Kurdakov** [comment]
there was already a link https://github.com/teremich/michis-mod-bot (that uses another api )
So livestream uses different api,
but it might be possible to make something along  https://github.com/teremich/michis-mod-bot approach using another api and 
adding those features which are already are in YT-spammer purge,
just it looks due to different API it might be a separate project

**Reply by KendallDoesCoding** [reply]
Ok, interesting..

**Comment by ThioJoe** [comment]
Hm for some reason I thought the API didn't have a way to access live chat messages, but it seems it can: https://developers.google.com/youtube/v3/live/docs/liveChatMessages

I'll definitely have to look into this, I know live chats can potentially be a spammer haven.
